python/ray/serve/tests/test_reconfiguration.py

```diff
@@ -0,0 +1,36 @@
+import pytest
```

Comment:
Rather than make a new file with a single test, should we put this test alongside the other `reconfigure` tests in `test_deploy.py`?  It would be great to use the `serve_instance, use_handle` fixtures too, if it makes sense to do that.

Reply:
Or we could move the existing `reconfigure` tests from `test_deploy.py` to this new file, what do you think?

Reply:
> 

`use_handle` fixture may not make sense, it seems that using asynchronous HTTP request will cause test flakiness.